src/pkg/cli/debug.go

```diff
@@ -27,47 +27,57 @@ var (
 	patterns            = []string{"*.js", "*.ts", "*.py", "*.go", "requirements.txt", "package.json", "go.mod"} // TODO: add patterns for other languages
 )
 
-func InteractiveDebug(ctx context.Context, c client.FabricClient, p client.Provider, etag types.ETag, project *compose.Project, failedServices []string, loadError error) error {
+func InteractiveDebugDeployment(ctx context.Context, c client.FabricClient, project *compose.Project, p client.Provider, etag types.ETag, failedServices []string) error {
+	return interactiveDebug(ctx, c, p, etag, project, failedServices, nil)
+}
+
+func InteractiveDebugForLoadError(ctx context.Context, c client.FabricClient, project *compose.Project, loadError error) error {
+	return interactiveDebug(ctx, c, nil, "", project, nil, loadError)
+}
```

Comment:
Thanks for breaking out this scaffolding. I think we can mostly erode `interactiveDebug` into these two functions, but I assume you're under time pressure to merge this now